ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -312,10 +315,24 @@ object Config {
           success
       })
 
+      opt[Unit]("early-access-unsafe")
```

Comment:
can we change it to just early-access?

Reply:
```suggestion
      opt[Unit]("early-access")
```

Reply:
If we don't need to be consistent with sandbox then an option name that implies that it is about `lf` would be preferred (i.e., the expression "early access" is fairly generic).